### DISCUSSION TITLE
Tailwind CLI not installed correctly, dev build continuously breaks

### DISCUSSION THREAD

**Discussion by alienking-sol** [tailwindlabs/tailwindcss#8641] [Help] [answered]
I keep getting this error when refreshing my app in dev build. No idea why, I'm certain I installed the dependencies correctly?  

`cdn.tailwindcss.com should not be used in production. To use Tailwind CSS in production, install it as a PostCSS plugin or use the Tailwind CLI: https://tailwindcss.com/docs/installation`

Whenever I make even the slightest change in a component, like adding a margin, this happens:
![image](https://user-images.githubusercontent.com/87619885/173700568-015f460b-da23-452b-a92c-6e314a66eba1.png)

I have to refresh about 4-5 times before it goes back to normal and I can see my change: 
![image](https://user-images.githubusercontent.com/87619885/173700656-ce56d67f-f5ec-4d8a-ad4c-82a79a4b93ac.png)

I'm super stuck and cannot figure it out. I've triple checked my dependencies, version and that I have the CLI correctly installed.

**Comment by rohmann** [comment, marked as answer]
Hi @alienking-sol,

Unfortunately these environment issues are nearly impossible to troubleshoot without access to something to reproduce it, but I've got a few thoughts that might be helpful:

* You shouldn't actually be using the CLI and the CDN at the same time.
* The CDN is great for experimenting and playing around, but for real projects you'll want to install and configure Tailwind as part of that project.
* The CLI is best if you're using pure HTML/CSS or otherwise don't have any bundling tools in play.
* If you're using a framework like React, you will want to follow the instructions for that: https://tailwindcss.com/docs/installation/framework-guides
* It might help to [start with a fresh Vite project](https://tailwindcss.com/docs/guides/vite) and compare that to your current setup.

Good luck getting it all sorted out!

**Reply by alienking-sol** [reply]
Hey @rohmann thank for the reply! I sorted it out :)

I had both the CLI and CDN installed. I removed CDN, followed the guide [here ](https://tailwindcss.com/docs/guides/nextjs) for Nextjs, added the correct paths in my [content], and all is groovy now!

appreciated it

**Reply by rohmann** [reply]
Ok great! Glad to hear it!
